roles/python-app/defaults/main.yml

```diff
@@ -2,5 +2,6 @@
 python_app_venv_path: /opt/venvs
 python_app_source_path: /opt/source
 
+python_app_git_repo: False
```

Comment:
Since this defaults to False, do you need to update some calls to this role that depend on this functionality?